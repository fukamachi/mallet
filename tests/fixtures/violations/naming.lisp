;; Test fixtures for naming convention rules

;; Bad: special variable without earmuffs
(defvar my-config "default")

;; Good: special variable with earmuffs
(defvar *my-config* "default")

;; Bad: defparameter without earmuffs
(defparameter debug-mode t)

;; Good: defparameter with earmuffs
(defparameter *debug-mode* t)

;; Bad: constant without plus signs
(defconstant pi-value 3.14159)

;; Good: constant with plus signs
(defconstant +pi-value+ 3.14159)

;; Bad: define-constant without plus signs
(alexandria:define-constant max-size 1000)

;; Good: define-constant with plus signs
(alexandria:define-constant +max-size+ 1000)
