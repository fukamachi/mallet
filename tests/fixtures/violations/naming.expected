# Expected violations for naming.lisp
# Format: line:column rule-name severity

4:8 special-variable-naming convention
10:14 special-variable-naming convention
16:13 constant-naming info
22:28 constant-naming info
